sonar-html-plugin/pom.xml

```diff
@@ -89,16 +74,16 @@
       <artifactId>sonar-analyzer-commons</artifactId>
       <version>${analyzerCommons.version}</version>
     </dependency>
+    <dependency>
+      <groupId>org.sonarsource.analyzer-commons</groupId>
+      <artifactId>sonar-analyzer-recognizers</artifactId>
+      <version>${analyzerCommons.version}</version>
+    </dependency>
     <dependency>
       <groupId>com.google.guava</groupId>
       <artifactId>guava</artifactId>
       <version>15.0</version>
     </dependency>
-    <dependency>
```

Comment:
This is used by IllegalElementCheck (among many others) and should not be removed.

Reply:
Fixed in 4581e29